### DISCUSSION TITLE
Any Support for VMb4500r2?

### DISCUSSION THREAD

**Discussion by shadowttw** [brianschrameck/arlo-cam-api#8] [Ideas]
As I am using VMb4500r2 for Arlo, I try to follow the guide but still cannot get the Base Station WPA-PSK

**Comment by brianschrameck** [comment]
Are you seeing any errors? Where in the process are you? What's your setup?

Have you tried just connecting the cameras to your own access point using WPA and not cloning the existing base station?

**Reply by shadowttw** [reply]
thx your reply, I try to use the ubuntu linux to capture the Real Base Station.

My wpa.config
<img width="389" alt="image" src="https://github.com/user-attachments/assets/eaa915df-cd55-4b67-92fb-435f9e373fcb">

located to desktop and my wifi USB info
<img width="404" alt="image" src="https://github.com/user-attachments/assets/0c793ed6-2e33-45be-9e1f-f73ea6dee8be">

I start the command hv the Invalid argument wanning
<img width="415" alt="image" src="https://github.com/user-attachments/assets/d75c6929-9d9d-43e3-91e2-4f5f3e37cd3d">

OMG! it's works now!! When I try to repeat my steps to you
<img width="434" alt="image" src="https://github.com/user-attachments/assets/f4854a0c-c516-438a-bc1f-711916e8ef8f">

**Reply by jsnyder** [reply]
I've tried something similar from an Intel NUC and if I try wps_pbc the wpa_supplicant log just shows WPS-PBC-ACTIVE and does nothing else until it times out.

I've also tried a  wps_pin and that does actually look like there's some back and forth with the base station, but that fails.  A scan in WPA Supplicant does show the AP, and that it supports WPS.  Should it list WPS-PBC if the Base Station has actually been switched into that mode? (I've not seen it change to that)

I've tried pressing the sync button and starting the "add a camera" process which I presume is what one would need to do in the app (where it searches for the camera) and pressing the button on the base station only very briefly changes the solid blue to the flashing light.  I'm wondering if the base station actually gets into the right mode?  I've tried rebooting it which doesn't seem to make a difference.

Thanks in advance if you have any ideas.

**Reply by jsnyder** [reply]
Switched from doing this on an Intel NUC to a raspberry Pi 5 and it worked without a hitch.  Wonder if there's something else about the chipset that the base station cares about or is looking for?  Should this work on non-broadcom chipsets?

**Reply by brianschrameck** [reply]
> Switched from doing this on an Intel NUC to a raspberry Pi 5 and it worked without a hitch.  Wonder if there's something else about the chipset that the base station cares about or is looking for?  Should this work on non-broadcom chipsets?

Hmm, interesting. I did this on an old HP laptop with an Intel 7260 WiFi chipset and it worked ok.

Maybe some weird OS/package version combo that doesn't work.

Glad to hear you got it up and running though!

**Reply by shadowttw** [reply]
> I've tried something similar from an Intel NUC and if I try wps_pbc the wpa_supplicant log just shows WPS-PBC-ACTIVE and does nothing else until it times out.
> 
> I've also tried a wps_pin and that does actually look like there's some back and forth with the base station, but that fails. A scan in WPA Supplicant does show the AP, and that it supports WPS. Should it list WPS-PBC if the Base Station has actually been switched into that mode? (I've not seen it change to that)
> 
> I've tried pressing the sync button and starting the "add a camera" process which I presume is what one would need to do in the app (where it searches for the camera) and pressing the button on the base station only very briefly changes the solid blue to the flashing light. I'm wondering if the base station actually gets into the right mode? I've tried rebooting it which doesn't seem to make a difference.
> 
> Thanks in advance if you have any ideas.

Ys, I know that feeling. I also have the same situation. But it was working suddenly when I reported this issue here.

**Reply by jsnyder** [reply]
> > Switched from doing this on an Intel NUC to a raspberry Pi 5 and it worked without a hitch.  Wonder if there's something else about the chipset that the base station cares about or is looking for?  Should this work on non-broadcom chipsets?
> 
> Hmm, interesting. I did this on an old HP laptop with an Intel 7260 WiFi chipset and it worked ok.
> 
> Maybe some weird OS/package version combo that doesn't work.
> 
> Glad to hear you got it up and running though!

Thanks! I figured there was some possibility this would make a difference even though it seemed like it maybe shouldn't.  It was admittedly a proxmox server (using the host environment which had no network manager running and nothing using the WiFi). It was just a Linux box that was already running that had WiFi and I could try it out. The Pi worked immediately on the first try. 

I think I've got my cameras sorted out now with some NAT rules in 1:1 and outbound in pfSense. Hoping Scrypted NVR does better with the powered cameras. I have 2 Pro 2s and 2 Pro 3s. I have a bunch of WiFi APs but trying to get decent video from one base station and it having to go to the cloud first was variable.  I might add in my doorbell on this as well. (Guess I could re-pair it to the base station then take that offline again).

**Comment by shadowttw** [comment]
After I setup the AP in my router and ready the arlo-local in scrypted. and started the arlo-cam-api docker in my NAS, but where can I add the device? I cannot see it in scrypted?

**Reply by brianschrameck** [reply]
Are you seeing camera messages in the logs of the API? What model cameras are they?

All you should have to do is add the URL to arlo-cam-api in Scrypted and it should populate the cameras automatically.

**Reply by shadowttw** [reply]
> Are you seeing camera messages in the logs of the API? What model cameras are they?
> 
> All you should have to do is add the URL to arlo-cam-api in Scrypted and it should populate the cameras automatically.

Thanks~! Finally, everything was fine. I do the port forwarding, and traffic route in the router setup and update the NAS docker's network to Host.
